packages/prompts/src/common.ts

```diff
@@ -3,8 +3,15 @@ import type { State } from '@clack/core';
 import isUnicodeSupported from 'is-unicode-supported';
 import color from 'picocolors';
 
-export const unicode = isUnicodeSupported();
-export const unicodeOr = (c: string, fallback: string) => (unicode ? c : fallback);
+export const env = {
```

Comment:
imo these are two unrelated exports

```ts
export const unicode = isUnicodeSupported();
export const isCI = () => process.env.CI === 'true';
```

Reply:
The advantages of the `env` object is the getter syntax that introduce less changes in the code (instead of `unicode`, it was just `env.unicode`) and it allows to unit test Unicode support (but only for spinner as all other usages are computed once).
And it is the same for `env.isCI`: with the getter syntax it is used as it was a simple property, where in reality it's a function. (Yet again, mainly to help unit testing)

---

But I can move these changes in another PR

Reply:
we don't mock them anywhere though, we mock what they rely on (e.g. `process.env`)

i can't see any functional reason for having them in this `env` object. i'd just export them individually

Reply:
in actual fact there'd be fewer changes that way too because all references to `unicode` would stay as they are, no?

Reply:
yes, because it is an ES module export like all the others we have

but we don't need to mock it anywhere. were you trying to make it mockable in preparation for new tests?